libs/core/src/lib/table/table.component.ts

```diff
@@ -114,4 +122,46 @@ export class TableComponent implements AfterViewInit, OnInit, OnDestroy {
             this._tableService.changeKeys([...keys]);
         }
     }
+
+    /** @hidden being triggered by TableCellDirective whenever it's clicked */
+    _onCellKeydown(event: KeyboardEvent, cell: TableCellDirective): void {
```

Comment:
@platon-rov changes for the platform table weren't merged. Moreover, since platform table is based on the core's implementation, this navigation thing will apply to the platform table by itself